### ISSUE TITLE
Rerank "Grazing Donkey"

### ISSUE THREAD

**Issue by JohanWiltink** [codewars/content-issues#65]
[Grazing Donkey](https://www.codewars.com/kata/grazing-donkey) is rather underrated.

I think the essential task justifies _at least_ a `5` and with the performance requirement certainly a `4`.

**Comment by Blind4Basics** [comment]
'dunno about 4 (I didn't solve it), but 5 at least, for sure.

**Comment by hobovsky** [comment]
The kata is not new, but by the years it was published it did not collect too many solves what might confirm the fact that it's harder than its rank suggests. IIRC, it got approved shortly after being published, and rank suggestions _could_ (but did not have to) be skewed by the votes coming from users known from voting 8 kyu on everything. The idea to re-rank appeared now because the kata resurfaced after receiving new translations. 

From the kata author (me): When the kata got approved, I was indeed surprised by the rank it received. When publishing, I hesitated between 4 and 5 kyu, finally estimated it as 5 kyu and it had no performance requirements then (large inputs were added after approval). But I was new to Codewars back then, so I just assumed people there are smart and this is the rank the kata deserves. I am not really bothered by the kata being misranked as long as it does not raise complaints. Apparently, math kata do not attract users who'd complain about kata being too difficult :)
